cmd/argocd/commands/project.go

```diff
@@ -609,35 +609,35 @@ func modifyResourceListCmd(cmdUse, cmdDesc string, clientOpts *argocdclient.Clie
 			}
 		},
 	}
-	command.Flags().StringVarP(&listType, "list", "l", defaultList, "Use blacklist or whitelist. This can only be 'white' or 'black'")
+	command.Flags().StringVarP(&listType, "list", "l", defaultList, "Use block list or allow list. This can only be 'allow' or 'block'")
```

Comment:
This help isn't technically correct as the code should still accept the old values to retain compatibility.